dvc/remote/gdrive.py

```diff
@@ -272,10 +282,16 @@ def get_remote_item(self, name, parents_ids):
 
         query += " and trashed=false and title='{}'".format(name)
 
+        param = {
+            "q": query,
+            "maxResults": 1,
```

Comment:
why do we limit it here? I would put an assert if it returns >1 items in the result

Reply:
it's related to the issue regarding duplicated folders with the same title.

Reply:
it is optimization to request and process only first value from the results set, but in case of duplicated folders issue it might return more than 1 result

Reply:
Makes sense! Resolving

Reply:
could you also update that comment a bit to be more explicit - like "Because of ... it's possible and allowed for remote to have two files with the same name">